wolfssl/wolfcrypt/port/Espressif/esp32-crypt.h

```diff
@@ -694,20 +769,25 @@ extern "C"
     WOLFSSL_LOCAL int esp_sha_hw_unlock(WC_ESP32SHA* ctx);
 
     /* esp_sha_hw_islocked: returns 0 if not locked, otherwise owner address */
-    WOLFSSL_LOCAL int esp_sha_hw_islocked(WC_ESP32SHA* ctx);
+    WOLFSSL_LOCAL uintptr_t esp_sha_hw_islocked(WC_ESP32SHA* ctx);
+
+    /* esp_sha_hw_in_use returns 1 (true) if SHA HW in use, otherwise 0 */
+    WOLFSSL_LOCAL int esp_sha_hw_in_use();
```

Comment:
When a function has not arguments please use `WOLFSSL_LOCAL intesp_sha_hw_in_use(void)`. Several other places too.

Reply:
Ah yes, good. An oversight on my part. Thanks for catching this.